ExtraEnemyCustomization/Patches/Handlers/FlyerStuckHandler.cs

```diff
@@ -20,7 +20,8 @@ public FlyerStuckHandler(IntPtr ptr) : base(ptr)
         {
         }
 
-        private void Update()
+#pragma warning disable IDE0051 // Remove unused private members
```

Comment:
Just a note: you can make the methods public or internal, or you can keep this.

Reply:
maybe making class itself internal would makes sense, I'll apply that after this PR has merged